### PR TITLE
Microverse Project: Ruby Bubble Sort

### PR DESCRIPTION
**Pull request by Janluc**
## Ruby Bubble Sort

- Created two different Ruby functions that are used for Sorting, one using yield and the other without using yield.

### CODE REVIEW

File: bubble.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    length_of_arr.times do |item|
+      if arr[item] > arr[item + 1]
+        arr[item], arr[item + 1] = arr[item + 1], arr[item]
+        swap = true
+      end
+    end
+
+    break unless swap
+  end
+  puts arr
```
Comment (maelfosso) [inline bubble.rb]: Returns the array don't print it.

File: bubble.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      value = yield(arr[item], arr[item + 1])
+      if value >= 1
+        arr[item], arr[item + 1] = arr[item + 1], arr[item]
+        swap = true
+      end
+    end
+
+    break unless swap
+  end
+  puts arr
```
Comment (maelfosso) [inline bubble.rb]: Returns the array don't print it.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,

Well done job done so far.
However, there is a small bug. 
Kindly read the comments left below, update your code and submit a new code review request

Thanks

Mael FOSSO

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations team, this project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:


Do not forget to fill the project completion form.
Again, congratulations, you can merge to master.
Good luck with your next task.

**I'm @taofeekolalere on slack.
Happy Coding :man_technologist:**

**Pull request merged**